Doc/tutorial/inputoutput.rst

```diff
@@ -433,10 +433,12 @@ an opaque number when in text mode.
 
 To change the file object's position, use ``f.seek(offset, whence)``.  The position is computed
 from adding *offset* to a reference point; the reference point is selected by
-the *whence* argument.  A *whence* value of 0 measures from the beginning
-of the file, 1 uses the current file position, and 2 uses the end of the file as
-the reference point.  *whence* can be omitted and defaults to 0, using the
-beginning of the file as the reference point. ::
+the *whence* argument.
+A *whence* value of :data:`io.SEEK_SET` measures from the beginning of the file,
+:data:`io.SEEK_CUR` uses the current file position,
+and :data:`io.SEEK_END` uses the end of the file as the reference point.
+*whence* can be omitted and defaults to :data:`!io.SEEK_SET`,
+using the beginning of the file as the reference point. ::
```

Comment:
I do find it strange that the tutorial delves into weird low-level APIs like `seek`. I can't even remember the last time I needed `seek()` for a real use case.

Reply:
I think it's reasonable to show that it's possible, especially after covering `tell`.

Reply:
Yes, if we introduce `tell`, surely we must also talk about `seek`. But I think an "how to do advanced I/O" is more fitting for these APIs.

Reply:
Well, conceptually it's not really advanced nor is it difficult to understand.

Reply:
The concept is easy enough to grasp. The APIs are a pain, though.